apps/src/templates/sectionProgressV2/IconKey.jsx

```diff
@@ -7,23 +7,30 @@ import AssignmentCompletionStatesBox from './AssignmentCompletionStatesBox';
 import styles from './progress-table-legend.module.scss';
 import {Heading6} from '@cdo/apps/componentLibrary/typography';
 import FontAwesome from '../FontAwesome';
+import Link from '@cdo/apps/componentLibrary/link';
+import MoreDetailsDialog from './MoreDetailsDialog.jsx';
 
 export default function IconKey({isViewingValidatedLevel, expandedLessonIds}) {
   const [isOpen, setIsOpen] = useState(false);
+  const [isIconDetailsOpen, setIconDetailsOpen] = useState(false);
+
+  const toggleIsViewingDetails = event => {
+    event.preventDefault();
+    setIconDetailsOpen(true);
+  };
 
   const isViewingLevelProgress = expandedLessonIds.length > 0;
 
   const caret = isOpenA => (isOpenA ? 'caret-down' : 'caret-right');
 
-  // TO-DO (TEACH-801): Fix spacing between boxes once width of the page is expanded
   const sectionContent = () => (
-    <div>
+    <>
```

Comment:
I have been trying to reduce the `div`-o-rama in my PRs.  I saw this little trick from a PR I reviewed of Eric's and it seemed to make sense here too.